decentralized/dynamics.py

```diff
@@ -214,61 +215,84 @@ def __init__(self, dt, *args, **kwargs):
         # net rotor force
         f_z = sym.symbols('f_z')
 
-        x = sym.Matrix([o_x, o_y, o_z, psi, theta, phi, v_x,
```

Comment:
Even though it is faster to hardcode everything at the bottom, it's kinda impossible to understand how that actually gets computed. So at the sacrifice of a slightly higher startup cost, we should revert this __init__ back to the original written out thing.